### ISSUE TITLE
Unable to pass env vars to set mongodb uri

### ISSUE THREAD

**Issue by bizob2828** [OWASP/NodeGoat#173]
We use NodeGoat to test some of our code and we programmatically pass in mongodb uri via `MONGODB_URI` env var.  This worked just fine until #147 was merged.  The changed in https://github.com/OWASP/NodeGoat/pull/147/files#diff-5ad5fa7320dd1d0531493bee405d385b made the 2 env vars useless.

**Comment by ckarande** [comment]
@bizob2828 Thanks for updating about the issue. 

@UlisesGascon  this seems to be order that would allow respecting values in env variables first for non-test environments. What do you think?
```
var db =  process.env.MONGOLAB_URI || process.env.MONGODB_URI || 'mongodb://nodegoat:owasp@ds159217.mlab.com:59217/nodegoat';
db =  process.env.NODE_ENV === 'test' ?  'mongodb://localhost:27017/nodegoat' : db;
 ```

**Comment by UlisesGascon** [comment]
Closed in #174
